Application/EdFi.Admin.DataAccess/Contexts/UsersContext.cs

```diff
@@ -47,7 +47,11 @@ public static string UserIdColumn
         public IDbSet<Profile> Profiles { get; set; }
 
         public IDbSet<OdsInstance> OdsInstances { get; set; }
-        
+
+        public IDbSet<OdsInstanceContext> OdsInstanceContexts { get; set; }
+
+        public IDbSet<OdsInstanceDerivative> OdsInstanceDerivatives { get; set; }
```

Comment:
Add IDbSet<OdsInstanceDerivative> OdsInstanceDerivatives { get; set; } in [IUsersContext.cs]